### DISCUSSION TITLE
Debian 11 - LGSM quits when logging out from shell

### DISCUSSION THREAD

**Discussion by nintendo424** [GameServerManagers/LinuxGSM#3610] [General]
Previously, on Debian 10/Buster, I would ssh into my instance, run `./arkserver start` and be good to close my shell and play.
Now, it seems that doing that and exiting force closes the server.

**Comment by johnoclockdk** [comment]
Any logs

**Comment by nintendo424** [comment]
Unfortunately no, since it seems to be killing the script. The logs are just showing the normal running logs.

**Comment by h3o66** [comment]
are there any system logs that indicate something ?
A while ago i heard about a systemd issue that could have done this.
Maybe you run into a OOM case, that the server kills the process.
Please provide overall more info about your case.

**Comment by johnoclockdk** [comment]
@nintendo424 bump

**Comment by dgibbs64** [comment]
I don't think this is a bug in LinuxGSM. Moving to discussions.
